pkg/koordlet/runtimehooks/hooks/cpuset/cpuset.go

```diff
@@ -51,10 +52,10 @@ func (p *cpusetPlugin) Register() {
 	rule.Register(name, description,
 		rule.WithParseFunc(statesinformer.RegisterTypeNodeTopology, p.parseRule),
 		rule.WithUpdateCallback(p.ruleUpdateCb))
-	reconciler.RegisterCgroupReconciler(reconciler.ContainerLevel, sysutil.CPUSet, p.SetContainerCPUSetAndUnsetCFS,
-		"set container cpuset and unset container cpu quota if needed")
-	reconciler.RegisterCgroupReconciler(reconciler.PodLevel, sysutil.CPUCFSQuota, UnsetPodCPUQuota,
-		"unset pod cpu quota if needed")
+	reconciler.RegisterCgroupReconciler(reconciler.ContainerLevel, []apiext.QoSClass{apiext.QoSLSE, apiext.QoSLSR},
+		sysutil.CPUSet, p.SetContainerCPUSetAndUnsetCFS, "set container cpuset and unset container cpu quota if needed")
+	reconciler.RegisterCgroupReconciler(reconciler.PodLevel, []apiext.QoSClass{apiext.QoSLSE, apiext.QoSLSR},
```

Comment:
remember to check the internal implements on ack

Reply:
consider preserve `qos=None` for cpuset and forbidden `Batch+None` for batch resource